src/main/java/uk/gov/hmcts/ccd/data/casedetails/search/SearchQueryFactoryOperation.java

```diff
@@ -77,7 +80,11 @@ private String secure(String clauses, MetaData metadata, Map<String, Object> par
     }
 
     private String addUserCaseAccessClause(Map<String, Object> params) {
-        if (UserAuthorisation.AccessLevel.GRANTED.equals(userAuthorisation.getAccessLevel())) {
+        if (applicationParam.getEnableAttributeBasedAccessControl()) {
```

Comment:
It would be to reflect this in tests. Not sure if you have manually tested this to prove case_ids is used in the query and works.